doc/sphinx/source/conf.py

```diff
@@ -46,7 +46,8 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.todo',
     'sphinx.ext.viewcode',
-    'sphinxcontrib.bibtex']
+    'sphinxcontrib.bibtex',
+    'sphinxcontrib.rsvgconverter']
```

Comment:
@jedbrown I've added the `rsvgconverter` extension and tested the `latexpdf` builder again, and the problem of the SVG files is gone. But now we have another error in the `latexpdf` builder, which I do not know how to solve. 
```
! Misplaced \omit.
\math@cr@@@ ...@ \@ne \add@amps \maxfields@ \omit 
                                                  \kern -\alignsep@ \iftag@ ...
l.1147 \end{align}\end{split}
```
It refers to `\omit` which we do not use directly in our latex/mathjax directives, So it must be internally generated somehow.